pkgs/applications/editors/quartus-prime/quartus.nix

```diff
@@ -0,0 +1,96 @@
+{ stdenv, lib, unstick, requireFile,
+  supportedDevices ? [ "Arria II" "Cyclone V" "Cyclone IV" "Cyclone 10 LP" "MAX II/V" "MAX 10 FPGA" ] }:
```

Comment:
```suggestion
{ stdenv, lib, unstick, requireFile
, supportedDevices ? [ "Arria II" "Cyclone V" "Cyclone IV" "Cyclone 10 LP" "MAX II/V" "MAX 10 FPGA" ]
}:
```

Reply:
Done, as well as the same change in `default.nix`.